jujugui/static/gui/src/app/components/status/status.js

```diff
@@ -5,23 +5,363 @@
 /** Status React component used to display Juju status. */
 class Status extends React.Component {
 
+  /**
+    Render the component.
+  */
   render() {
     return (
       <juju.components.Panel
         instanceName="status-view"
         visible={true}>
         <div className="status-view__content">
-          Status
+          {this.renderStatus()}
         </div>
       </juju.components.Panel>
     );
   }
 
-};
+  /**
+    Render the current model status.
+  */
+  renderStatus() {
```

Comment:
I would reply to @hatched that all those methods are called by render in order to do the render job, which is rendering elements. The render prefix makes it clear that these methods are used in the rendering process.